pkg/controllers/resources/csinodes/syncer.go

```diff
@@ -0,0 +1,48 @@
+package csinodes
+
+import (
+	"github.com/loft-sh/vcluster/pkg/controllers/syncer"
+	synccontext "github.com/loft-sh/vcluster/pkg/controllers/syncer/context"
+	"github.com/loft-sh/vcluster/pkg/controllers/syncer/translator"
+	storagev1 "k8s.io/api/storage/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func New(ctx *synccontext.RegisterContext) (syncer.Object, error) {
+	return &csinodeSyncer{
+		Translator: translator.NewMirrorPhysicalTranslator("csinode", &storagev1.CSINode{}),
+	}, nil
+}
+
+type csinodeSyncer struct {
+	translator.Translator
+}
+
+// TODO, only sync selected nodes
```

Comment:
Would be nice to create an issue about this and link it from the comment if you are going to do other edits.
This issue could probably be marked as a "good-first-issue", we don't have many of those, so it would be nice.

Reply:
I should get this into this PR, this was a TODO for me :) Good catch!

Reply:
Just updated the PR, but forgot this. Will update.

Reply:
Updated the PR with this :)